financials/spending_log.csv

```diff
@@ -1,2 +1,3 @@
 spend_id,date,spend_description,decision_log_id,currency,amount,exchange_rate,USD_equivalent,from_address,to_address,tx_reference,comment
-1,20190301,Coinspect audit fee,19,BTC,4.482,3882.66,17402.08,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,17Du5PQfcLq1yeV8yw512M5UnkeZvTwe7f,https://www.blockchain.com/btc/tx/022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468,30% advance payment
\ No newline at end of file
+1,20190301,Coinspect audit fee,19,BTC,4.482,3882.66,17402.08,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,17Du5PQfcLq1yeV8yw512M5UnkeZvTwe7f,https://www.blockchain.com/btc/tx/022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468,30% advance payment
+2,20190314,Igno donation,3 months funding for Igno's work,?,GRIN,9593.296153967,2.70,25901.9,https://donation.grin-tech.org,-,1ed0e7c7-6f15-4ed6-97f2-b1c16a6af7f3,not enough grins for full 30k payment so rest to be completed when possible

```

Comment:
I'm getting an error when trying to view this `.csv`: "It looks like row 3 should actually have 12 columns, instead of 13."